bittensor/_axon/__init__.py

```diff
@@ -35,90 +35,92 @@
 from . import axon_impl
 
 class axon:
-    """ The factor class for bittensor.Axon object
-    The Axon acts a grpc server for the bittensor network and allows for communication between neurons.
-    By default, the grpc server follows the bittensor protocol and transports forward and backwards requests
-    between validators and servers. 
+    """ The factory class for bittensor.Axon object
+    The Axon is a grpc server for the bittensor network which opens up communication between it and other neurons.
+    The server protocol is defined int bittensor.proto and describes the manner in which forward and backwards requests
```

Comment:
this is my mistake, but ive added a typo here " is defined in" not " is defined int "

Reply:
Thanks ! solved it